test/rules/no-absolute-path.spec.ts

```diff
@@ -66,72 +62,72 @@ ruleTester.run('no-absolute-path', rule, {
   ],
   invalid: [
     tInvalid({
-      code: `import f from "${absolutePath('/foo')}"`,
-      filename: absolutePath('/foo/bar/index.js'),
+      code: `import f from "/foo"`,
+      filename: '/foo/bar/index.js',
       errors: [ABSOLUTE_ERROR],
       output: 'import f from ".."',
     }),
     tInvalid({
-      code: `import f from "${absolutePath('/foo/bar/baz.js')}"`,
```

Comment:
I don't quite understand this change, it doesn't cover Windows absolute paths.

Reply:
It's not possible to do an absolute path import on Windows - or at least, I couldn't work out how to make one.

Reply:
https://github.com/nodejs/node/issues/31710

`file://` protocol can be used for Windows absolute path, but of course we maybe not handling `file://` protocol previously, same as other protocols like `http://` or `https://`.

But in case that `file://` is only for absolute local path like previous, it could be an improvement.

Reply:
Yes, using a file protocol path seems like a different question to a posix absolute path - or at least an additional feature to be added to this rule.